packages/components/addon/components/hds/dropdown/toggle.js

```diff
@@ -0,0 +1,114 @@
+import Component from '@glimmer/component';
```

Comment:
So this was a template-only component because it's abstracted away from the developer (and as such, it doesn't need so many guards). As a template-only component it will contribute to faster build times, too.  Let's think about whether or not we really need to do add a backing class. I don't think we do.

Reply:
See comments below about end-user performance (the most important one). Re. build time, for such small JS files there's definitely no perceivable difference, I'm pretty confident.